src/App/Application.cpp

```diff
@@ -457,19 +457,19 @@ void Application::renameDocument(const char *OldName, const char *NewName)
 Document* Application::newDocument(const char * Name, const char * UserName, DocumentCreateFlags CreateFlags)
 {
     auto getNameAndLabel = [this](const char * Name, const char * UserName) -> std::tuple<std::string, std::string> {
-        bool defaultName = (!Name || Name[0] == '\0');
+        bool isDefaultName = Base::Tools::isNullOrEmpty(Name);

```

Comment:
No need to explicitly write `Base::` when we have `using namespace Base` at the top
```suggestion
        bool isDefaultName = Tools::isNullOrEmpty(Name);
```